### PR TITLE
[Event] Remove usages of `sponsorship_fee` column and drop it

### PR DESCRIPTION
**Pull request by rluodev**
this step: drop column

next step: unignore column

### CODE REVIEW

File: db/schema.rb
Comment (sampoder) [inline db/schema.rb:1]: Could you drop the unrelated changes?

Comment (rluodev) [reply db/schema.rb:1]: oop yes

Comment (rluodev) [reply db/schema.rb:1]: All done!

**Review by sampoder — APPROVED** [review]

**Comment by garyhtou** [comment]
🪄

No need for review from me on this!
